openshift/CLOUD-2262.adoc

```diff
@@ -258,11 +258,17 @@ This should be considered as a new feature request.
 
 === Hard Requirements
 
-* EJB remote calls have to work when application is deployed on JBoss EAP on OpenShift
-* The transaction consistency has to be guaranteed for the transactions propagated
-  over the EJB remote calls
+* allow users to deploy clustered transactional EAP applications on OpenShift
```

Comment:
will we define in the requirements that the stateful set will be used or is it rather an implementation detail?

Reply:
not needed  :)